modules/syslogformat/syslog-format.c

```diff
@@ -1037,6 +754,12 @@ log_msg_parse_legacy(const MsgFormatOptions *parse_options,
 
   log_msg_parse_seq(self, &src, &left);
   log_msg_parse_skip_chars(self, &src, &left, " ", -1);
+
+  if (!log_msg_extract_cisco_timestamp_attributes(self, &src, &left, parse_options->flags))
+    {
+      goto error;
```

Comment:
Good point. I've modified the call-site so that it ignores return value, just like with log_msg_parse_seq(), which also has an ignored return value.

The last two patch I've just pushed should unify the conventions here.